lib/Backend/CodeGenWorkItem.h

```diff
@@ -42,7 +42,7 @@ struct CodeGenWorkItem : public JsUtil::Job
 
     uint GetFunctionNumber() const
     {
-        return this->jitData.jitData->bodyData->funcNumber;
+        return this->functionBody->GetFunctionNumber();
```

Comment:
>this->functionBody [](start = 15, length = 18)

What the difference between this->functionBody and this->jitData.jitData->bodyData? Also wondering why we could get AV in the latter case.